src/controls/ProgressStepsIndicator/ProgressStepsIndicatorStyles.ts

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import {
+  ILabelStyles,
+  IStackStyles,
+  IStyle,
+  mergeStyleSets,
+} from 'office-ui-fabric-react';
```

Comment:
Import from lib to align with the rest of the project, please :)
office-ui-fabric-react/lib/Label
office-ui-fabric-react/lib/Stack
office-ui-fabric-react/lib/Styling

Reply:
fixed